kedro-airflow/tests/test_plugin.py

```diff
@@ -263,6 +268,31 @@ def test_create_airflow_dag_env_parameter_exists(cli_runner, metadata):
         dag_code = [line.strip() for line in f.read().splitlines()]
     assert expected_airflow_dag in dag_code
 
+    _kedro_remove_env(Path.cwd())
+
+
+def test_create_airflow_dag_tags_parameter_exists(cli_runner, metadata):
+    """Test the `tags` parameter"""
+    dag_name = "hello_world"
+    command = ["airflow", "create", "--tags", "tag0", "--env", "remote"]
+
+    _kedro_create_env(Path.cwd())
+
+    dag_file = Path.cwd() / "airflow_dags" / f"{dag_name}_remote_dag.py"
+    result = cli_runner.invoke(commands, command, obj=metadata)
+
+    assert result.exit_code == 0, (result.exit_code, result.stdout)
+    assert dag_file.exists()
+
+    expected_airflow_dag = 'tasks["node0"] >> tasks["node2"]'
```

Comment:
I would suggest moving test parameters to `@pytest.mark.parametrize` to keep the test cleaner.  However, I've looked through other tests and found they're organised similarly, so please ignore this comment if there's a specific intent to do this.

Reply:
Thank you. I moved them to `parametrize` while adding a new test that involves multiple DAGs. 
I think it's not organized to use numerous @pytest.mark.parametrize with the intention of providing a clear output that includes the test name, which specifies the exact problem, instead of put too many options into parametrize in a single test.